src/spritecache.cpp

```diff
@@ -31,18 +31,17 @@ uint _sprite_cache_size = 4;
 struct SpriteCache {
 	void *ptr;
 	size_t file_pos;
+	SpriteFile *file;    ///< The file the sprite in this entry can be found in.
```

Comment:
perhaps const? to more clearly show non-owning

Reply:
Only caveat is that making it const means it can't change its state, and reading bytes from the file actually changes the state of the object.

Reply:
Oh, I mean `SpriteFile * const` ... but eh.

Reply:
`	SpriteFile * const file;    ///< The file the sprite in this entry can be found in.` results in:
src\spritecache.cpp(635): error C2166: l-value specifies const object
src\spritecache.cpp(652): error C2166: l-value specifies const object

`	const SpriteFile *file;    ///< The file the sprite in this entry can be found in.` results, after passing the const to everyinth that is called with that in things like:
src\spriteloader\grf.cpp(254): error C2662: 'uint32 RandomAccessFile::ReadDword(void)': cannot convert 'this' pointer from 'const SpriteFile' to 'RandomAccessFile &'